src/components/Image/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ImgHTMLAttributes, useState } from 'react';
+import { styled } from '@mui/system';
+import { Container } from '@mui/material';
+
+interface ImageDetails {
+  imageCustomise?: ImgHTMLAttributes<HTMLImageElement>;
+  size?: 'extra-small' | 'small' | 'medium' | 'large' | 'extra-large';
+  avatar: boolean;
```

Comment:
should this be required?

Reply:
After extending the interface it's not allowing me to leave it as optional